waspc/examples/todo-typescript/node_modules/@wasp/api/index.ts

```diff
@@ -0,0 +1,110 @@
+import axios, { type AxiosError } from "axios";
+
+import config from "@wasp/core/config";
+import { storage } from "@wasp/core/storage";
+import { apiEventsEmitter } from "@wasp/api/events";
```

Comment:
> so how does having this code in separate packages make development faster for us? **I would imagine we are still generating all of this code via Generator** [...]
> The only pro I am not sure about is the effect on our development speed -> I would like to understand that better.

We wouldn't be generating this (template-independent) code using the generator.

Since there's nothing dynamic about those files, we could maintain them as proper NPM packages (without any templating or code generation) and test them as such.

The generator's only job would then become choosing which packages to include in the user's project.

Reply:
We will be going with a single package for now since it's more natural for the users.

For our internal development, we plan to modularize the code in a way that ensures possible testing of the code from group 1 and 2 independently of the code in group 3.

Reply:
I've updated the main SDK organization issue (#1429) with a link to this disucssion.